API/API.mli

```diff
@@ -5949,8 +5946,15 @@ sig
 
   type deprecation = bool
 
+  type atts = {
+    loc : Loc.t option;
+    locality : bool option;
```

Comment:
Not sure it is meaningful to special-case locality in a future world where we have attributes, as it's a part of an effect (e.g. local definition but not local instance and conversely). But I agree that we'll have to do that progressively.

Reply:
Yeah this commit is not very ambitious in that regard as it doesn't mean to interfere with the people working on attributes but do the minimum to remove the `LocalityFixme` hack.

Reply:
This first step is consistent with the attributes plan AFAICT.